packages/main/src/SideNavigationItem.js

```diff
@@ -0,0 +1,251 @@
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import {
+	isTabNext,
+	isTabPrevious,
+	isDown,
+	isLeft,
+	isUp,
+	isRight,
+} from "@ui5/webcomponents-base/dist/Keys.js";
+import Title from "./Title.js";
+import Icon from "./Icon.js";
+import "@ui5/webcomponents-icons/dist/icons/navigation-down-arrow.js";
+import "@ui5/webcomponents-icons/dist/icons/navigation-right-arrow.js";
+import SideNavigationItemTemplate from "./generated/templates/SideNavigationItemTemplate.lit.js";
+
+// Styles
+import SideNavigationItemCss from "./generated/themes/SideNavigationItem.css.js";
+
+/**
+ * @public
+ */
+const metadata = {
+	tag: "ui5-side-navigation-item",
+	managedSlots: true,
+	properties: /** @lends sap.ui.webcomponents.main.SideNavigationItem.prototype */ {
+		/**
+		 * Defines the text of the item.
+		 *
+		 * @public
+		 * @type {String}
+		 */
+		text: {
+			type: String,
+		},
+
+		/**
+		 * Defines the icon of the item.
+		 *
+		 * @public
+		 * @type {string}
+		 */
+		icon: {
+			type: String,
+		},
+
+		/**
+		 * Defines whether the item is expandable. In order to use this property you need to pass another <code>ui5-side-navigation-item</code> as a default slot.
+		 * @public
+		 * @type {Boolean}
+		 */
+		expandable: {
+			type: Boolean,
+		},
+
+		/**
+		 * If <code>expandable</code> is set to <code>true</code>, you can control via this property whether the current item is expanded or not.
```

Comment:
`Defines if the item is expanded`
`<br><br>`
`<b>Note:</b>`(and put your text in note)